tests/test_asset_connect.py

```diff
@@ -40,7 +40,7 @@ def test_delivery_connection(self):
 
         conn = S3Connection()
         try:
-            bucket = conn.get_bucket(self.settings['aws_deliver_bucket'])
+            bucket = conn.get_bucket(self.settings['veda_deliverable_bucket'])
```

Comment:
settings does not have any thing named `veda_deliverable_bucket`, we've to define it.